entrypoint.sh

```diff
@@ -2,21 +2,38 @@
 
 set -Euo pipefail
 
+OPA_ROOT_TOKEN=$(cat /run/secrets/opa-root-token)
+
 if [[ -f "/app/initial_setup" ]]; then
     sed -i s/CLIENT_ID/$KEYCLOAK_CLIENT_ID/ /app/permissions_engine/idp.rego && sed -i s/CLIENT_ID/$KEYCLOAK_CLIENT_ID/ /app/permissions_engine/authz.rego
     sed -i s/OPA_SITE_ADMIN_KEY/$OPA_SITE_ADMIN_KEY/ /app/permissions_engine/idp.rego && sed -i s/OPA_SITE_ADMIN_KEY/$OPA_SITE_ADMIN_KEY/ /app/permissions_engine/authz.rego
 
     OPA_SERVICE_TOKEN=$(cat /run/secrets/opa-service-token)
     sed -i s/OPA_SERVICE_TOKEN/$OPA_SERVICE_TOKEN/ /app/permissions_engine/authz.rego
 
-    OPA_ROOT_TOKEN=$(cat /run/secrets/opa-root-token)
     sed -i s/OPA_ROOT_TOKEN/$OPA_ROOT_TOKEN/ /app/permissions_engine/authz.rego
-    echo "initializing idp"
-    python3 /app/permissions_engine/initialize_idp.py
-    rm /app/initial_setup
+
+    echo "initializing stores"
+    python3 /app/initialize_vault_store.py
+    if [[ $? -eq 0 ]]; then
+        rm /app/initial_setup
+        rm /app/bearer.txt
+        echo "!!!!!! SETUP COMPLETE !!!!!!"
+    else
+        echo "initialization failed, try again"
```

Comment:
The `!!! ALL CAPITALS !!!` makes more sense for a failed state than an "all ok" state -- might be a good idea to swap these two
```suggestion
        echo "setup complete"
    else
        echo "!!!!!! INITIALIZATION FAILED, TRY AGAIN !!!!!!"
```

Reply:
In addition, I notice that /app/initialize_vault_store.py gives multiple exit codes, each meaning a different thing. It might make sense to be a bit verbose here:
```suggestion
        echo "setup complete"
    elif [[ $? -eq 1 ]]; then
        echo "!!!!!! /APP/BEARER.TXT COULD NOT BE READ: SEE LOGS FOR DETAILS !!!!!!"
    elif [[ $? -eq 2 ]]; then
        echo "!!!!!! ERROR WHILE SETTING ACCESS SECRET: SEE LOGS FOR DETAILS !!!!!!"
    elif [[ $? -eq 3 ]]; then
        echo "!!!!!! ERROR WHILE SETTING PATHS SECRET: SEE LOGS FOR DETAILS !!!!!!"
    else
        echo "!!!!!! UNKNOWN ERROR OCCURRED: SEE LOGS FOR DETAILS !!!!!!"
```